__tests__/server/utils/createCircuitBreaker.spec.js

```diff
@@ -132,13 +132,7 @@ describe('Circuit breaker', () => {
     setEventLoopDelayThreshold(-1);
     jest.advanceTimersByTime(5e3 + 10);
     await mockCircuitBreaker.fire('hola, mundo');
-    expect(consoleErrorSpy.mock.calls).toMatchInlineSnapshot(`
```

Comment:
Had to change this because the snapshot kept coming back as :

> [Error: Opening circuit, p(100) event loop delay (0.005112ms) is > eventLoopDelayThreshold (-1ms)],

Seems like a rounding difference, but didn't want to spend too much time figuring that out